libs/sharedcomponents/src/packages/sidesheet/tabs/CustomTabs.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useContext, createContext } from 'react';
+import { TabsProps, TabPanelsProps } from '@equinor/eds-core-react';
+
+import { StyledTabs, StyledPanels } from './tabs.styles';
+import React from 'react';
+
+// ########## Custom Tabs Context ##########
+
+const TabsContext = createContext(0);
+
+// ########## Custom Tabs ##########
+
+type CustomTabsProps = {
+    initialTabIndex?: number;
+}
+
+type CustomProps = CustomTabsProps & TabsProps
+
+export const CustomStyledTabs = (props: CustomProps) => {
```

Comment:
If this works without bugs and quirks. We should look into applying it to all the other apps. Not in this pr but in a new one.

Reply:
Yes, i think this works ok.
Noted, i can make a issue on it.